homeassistant/components/ring/light.py

```diff
@@ -26,8 +27,12 @@
 
 SKIP_UPDATES_DELAY = timedelta(seconds=5)
 
-ON_STATE = "on"
-OFF_STATE = "off"
+
+class OnOffState(StrEnum):
+    """Enum for allowed on off states."""
+
+    ON = auto()
+    OFF = auto()
```

Comment:
What's this?

Reply:
The `StrEnum` was so that I could type hint the possible state values.  The `auto()` sets the string value to the lower case value of the member name.